src/dvc_render/base.py

```diff
@@ -18,10 +18,10 @@ class Renderer(abc.ABC):
 
     EXTENSIONS: Iterable[str] = {}
 
-    def __init__(self, datapoints: Dict, name: str, **properties):
-        self.datapoints = datapoints
-        self.name = name
-        self.properties = properties
+    def __init__(self, datapoints: List, name: str, **properties):
+        self.datapoints = datapoints or []
+        self.name = name or ""
+        self.properties = properties or {}
```

Comment:
If there are no kwargs passed, it's an empty dict by default.
```suggestion
        self.properties = properties
```